Impossible-Agent-Suite/DangerousActivities/README.md

```diff
@@ -0,0 +1,19 @@
+# Impossible Finance Dangerous Activities
+
+## Description
+
+This agent detects transactions involving impossible addresses and dangerous addresses
+
+## Supported Chains
+
+- BSC
+
+## Alerts
+
+- impossible-dangerous
```

Comment:
According to the agent table in the Impossible Finance Notion page, this agent should be named `IMPOSSIBLE-8`
```suggestion
- IMPOSSIBLE-8
```